app/views/static_pages/formula.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="menu">
+  <a class="tab index tooltip active" href="/index" name="Projects">
+    <img src="<%= asset_path('leaf-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/attributes" name="Attributes">
+    <img src="<%= asset_path('edit-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/information" name="Information">
+    <img src="<%= asset_path('info-white.svg') %>">
+  </a>
+  <a class="tab logout tooltip" href="/.." name="Log Out">
```

Comment:
Cool no prob! Could you stick a `TODO` anyway? It won't hurt and there's a chance you'll copy and paste this stuff somewhere else.